oaknational/Oak-Web-Application
# Feat/lesq 1046/threads styling [LESQ-1046]

## Description

Music year: 1953

- Refactor unitsLearningThemeFilters to use radio buttons for threads instead of links
- Add a component RadioTile, will extract this to components as a separate ticket

## How to test

1. Go to https://deploy-preview-2784--oak-web-application.netlify.thenational.academy/teachers/programmes/chemistry-secondary-ks4-foundation-ocr/units
2. Click on Themes along the right
3. You should see the inputs toggle and units update

## Screenshots

How it used to look (delete if n/a):
<img width="600" alt="Screenshot 2024-09-17 at 08 13 19" src="https://github.com/user-attachments/assets/1a574b5c-e08c-45f4-8bb4-5a35d25b8648">

How it should now look:

<img width="600" alt="Screenshot 2024-09-17 at 08 43 29" src="https://github.com/user-attachments/assets/ebc2c00d-153f-4c89-a5a3-076aa00e45ff">

## Ticket

https://www.notion.so/oaknationalacademy/I-want-threads-to-be-restyled-as-radio-buttons-25be5006e6084a9a9986a261f8f9b54a?pvs=4
